Assets/Scripts/Commands/ServerGameState.cs

```diff
@@ -100,11 +83,11 @@ public override void Apply(Ffb.Dto.Commands.ServerGameState command)
                 player.Spp = p.currentSpps;
             }
 
-            positions.Clear();
             roster = command.game.teamAway.roster;
+            FFB.Instance.Model.PositionsAway.Clear();
             foreach (var pos in roster.positionArray)
             {
-                positions[pos.positionId] = new Position()
+               FFB.Instance.Model.PositionsAway[pos.positionId] = new Position()
```

Comment:
This should use the same method of constructing position instances as the players above.